src/track/keyutils.cpp

```diff
@@ -395,25 +395,25 @@ ChromaticKey KeyUtils::scaleKeySteps(ChromaticKey key, int key_changes) {
 
 // static
 mixxx::track::io::key::ChromaticKey KeyUtils::calculateGlobalKey(
-    const KeyChangeList& key_changes, const int iTotalSamples) {
+    const KeyChangeList& key_changes, const int iTotalSamples, const int iSampleRate) {
     const int iTotalFrames = iTotalSamples / 2;
-    QMap<mixxx::track::io::key::ChromaticKey, double> key_histogram;
+    QMap<mixxx::track::io::key::ChromaticKey, double> key_occurrence;
```

Comment:
This is still a *histogram* and not a *distribution* if you only normalize the sums for debug output (see below). Also the term *occurrence* doesn't fit here and doesn't tell me what it actually means.

Reply:
Ups, yes i was confused about what a histogram is.